<launch>

  <node pkg="rviz_cloud_annotation" type="rviz_cloud_annotation_node" name="rviz_cloud_annotation_node" output="screen">
    <param name="pcd_type" type="string" value="XYZI" /> <!-- supported pcd format: XYZ, XYZI, XYZRGB -->
    <param name="update_topic" type="string" value="/point_cloud" />
    <param name="LogFile" type="string" value="$(env HOME)/lidar_annotation/annotation.log" />
    <param name="cloud_file_folder" type="string" value="$(env HOME)/lidar_annotation/pcd/" />
    <param name="image_file_folder" type="string" value="$(env HOME)/lidar_annotation/image/" />
    <param name="annotation_file_folder" type="string" value="$(env HOME)/lidar_annotation/_annotation/" />
    <param name="annotation_cloud_folder" type="string" value="$(env HOME)/lidar_annotation/_pcd/" />
    <param name="bbox_names_file_folder" type="string" value="$(env HOME)/lidar_annotation/_bbox/" />
    <param name="label_names_file_folder" type="string" value="$(env HOME)/lidar_annotation/_label/" />
    <param name="line_names_file_folder" type="string" value="$(env HOME)/lidar_annotation/_line/" />

    <param name="label_size" type="double" value="0.2" />
    <param name="neigh_search_type" type="int" value="2" />
    <param name="neigh_search_params" type="string" value="10" />

    <param name="normal_importance" type="double" value="0.0" />
    <param name="color_importance" type="double" value="0.0" />
    <param name="position_importance" type="double" value="1.0" />

    <param name="show_zero_weight_control_points" type="bool" value="false" />
  </node>

  <node pkg="rviz" type="rviz" name="annotation_rviz" output="screen" args="-d $(find rviz_cloud_annotation)/rviz/annotation.rviz" required="true" />

</launch>
